foundation-extensions/identity-for-edge-network/api-reference.md

```diff
@@ -339,10 +339,22 @@ AEPIdentityItem *item = [[AEPIdentityItem alloc] initWithId:@"user@example.com"
 
 ## resetIdentities
 
-Clears all identities stored in the Identity extension and generates a new Experience Cloud ID \(ECID\) . Using this API does not remove the identifiers from the server-side User Profile Graph or Identity Graph.
+Clears all identities stored in the Identity extension and generates a new Experience Cloud ID (ECID). Using this API does not remove the identifiers from the server-side User Profile Graph or Identity Graph.
+
+This is a destructive action, since once an ECID is removed it cannot be reused. The new ECID generated by this API can impact metrics like unique visitors and cause duplication of user profiles and device profiles.
+
+Some example use cases for this API are: 
+* During debugging, to see how new ECIDs (and other identifiers paired with it) behave with existing rules and metrics.
+* A last-resort reset for when an ECID should no longer be used.
+
+This API is not recommended for:
+* Resetting a user's consent and privacy settings; see [Privacy and GDPR](../../resources/privacy-and-gdpr.md)/
+* Removing existing custom identifiers, use [`removeIdentity`](#removeidentity) API instead.
```

Comment:
nit: let's use same format here, semicolon before use.

Reply:
"... use the [`removeIdentity`] API instead."

Reply:
Got it, do we not use code styling for API links?

Reply:
Oh! The code styling is fine, just you'd want to add the the in front!

Reply:
Oh whoops i didnt even notice the "the"; I'll add that in